.github/workflows/test.yml

```diff
@@ -3,13 +3,27 @@ name: test
 on:
   push:
     branches:
-      - "*"
+      - master
+      - "test-me-*"
 
   pull_request:
     branches:
       - "*"
 
+# Cancel running jobs for the same workflow and branch.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
+
+  check-package:
```

Comment:
It would be nice to produce the package once, using this job, and then reuse this package for all the 'test' job, but I couldn't find an easy way to ask for `tox` to use a specific wheel to test (instead of building its own).

Reply:
Actually I just discovered the `--installpkg` from `tox run` which does exactly that.

Will try it out in a separate PR.